packages/ui/src/components/button-group/button-group.stories.tsx

```diff
@@ -39,10 +38,19 @@ const SIZES = ['small', 'medium', 'large', 'xlarge'] as const;
  */
 export const Default: Story = {
   args: {
-    children: [
-      <ButtonGroupButton value="Option 1">Option 1</ButtonGroupButton>,
-      <ButtonGroupButton value="Option 2">Option 2</ButtonGroupButton>,
-      <ButtonGroupButton value="Option 3">Option 3</ButtonGroupButton>,
+    buttons: [
+      {
+        value: 'Option 1',
+        children: 'Option 1',
```

Comment:
should this be label/value? or we children can be a react component too?

Reply:
I should be able to make this change, makes more sense to have it as a string call label

Reply:
I agree, since children in this case is not actually React children we should use a different name to avoid confusion. Fine to use your suggestion of label/value.

Reply:
@samithaf children here can be a react component since it just gets spread onto the item https://github.com/WestpacGEL/GEL-next/pull/702/files#diff-60f38f057bd071871bf69a97fb7c782f1ca226f42ed8f56741b586839e9ab65aR61
Do we see a need to open up the children to have an optional wrapper since we are now no longer exposing them? I agree that most cases wont need to and we should change it to label/value but I can see where @tomjackking was thinking in having it as children so that it was still open for devs to change the items as needed.

Reply:
@samithaf @tomjackking I would follow how they have done it in the react docs here https://react.dev/reference/react/Children#accepting-an-array-of-objects-as-a-prop and change it from children to label and have the type of label be ReactNode

Reply:
cool so, Type would be, label: string; value: React.ReactNode, _hint: React.ReactNode_ (hint applicable for some of the components)?

Reply:
@samithaf I believe the type would be value: string (the value of button(as a radio)/radio/checkbox for forms etc.), label: ReactNode (rename of children)

Reply:
Yep my mistake! Label needs the flexibility.